.experimenter.yaml

```diff
@@ -61,9 +61,6 @@ homescreenFeature:
   hasExposure: true
   exposureDescription: ""
   variables:
-    jump-back-in-synced-tab:
```

Comment:
This keeps popping up for me, so might as well commit this here. We removed this nimbus experiment recently in another PR.